lightning/src/util/ser_macros.rs

```diff
@@ -131,6 +146,9 @@ macro_rules! check_missing_tlv {
 			$field = $default.into();
 		}
 	}};
+	($last_seen_type: expr, $type: expr, $field: expr, (reset_on_reload, $value: expr)) => {
+		$field = $value;
+	};
```

Comment:
Could we call this `exclude` and use Default::default() instead of providing a value?

Reply:
I am not sure if i understand correctly, shouldn't this kind of thing be while reading or a constructor thing.